patches/master_patch.patch

```diff
@@ -1795,6 +1824,30 @@ index 24ba9fd2be824110b84ac82700989c2e86b48ffb..0f11dbc0e84c2d63f839fc086feab3e1
        "chrome://chrome-signin/*",
        "chrome://media-router/*",
        "chrome://mobilesetup/*",
+diff --git a/extensions/common/extension_urls.cc b/extensions/common/extension_urls.cc
+index 769fbc01794c2719d8a0d529d308274ecf15fd5b..c6ced6e9f4e5496276fb69b1d6af91a633b4ed77 100644
+--- a/extensions/common/extension_urls.cc
++++ b/extensions/common/extension_urls.cc
+@@ -103,11 +103,19 @@ bool IsBlacklistUpdateUrl(const GURL& url) {
+ }
+ 
+ bool IsSafeBrowsingUrl(const url::Origin& origin, base::StringPiece path) {
```

Comment:
please use ShouldHideBrowserNetworkRequest in AtomExtensionsAPIClient instead of patching here. The only place this is used is `IsSensitiveRequest` and that also checks ShouldHideBrowserNetworkRequest